.github/workflows/docker-build-and-test.yml

```diff
@@ -0,0 +1,104 @@
+name: Docker Build and Test
+
+on:
+  workflow_dispatch:
```

Comment:
Can you add branches here as well?
I know that you can select the branch manually.

Reply:
I will add this in next PR, I want this absolutely isolated to this branch for now.

Reply:
Yes, I mean can't we add the branch name here as you did below for the pull request event?